auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -403,17 +407,111 @@ public static AuthUI getInstance(FirebaseApp app) {
         return R.style.FirebaseUI;
     }
 
+
+    /**
+     * Configuration for an identity provider.
+     *
+     * In the simplest case, you can supply the provider ID and build the config like this:
+     * {@code new IdpConfig.Builder(AuthUI.GOOGLE_PROVIDER).build()}
+     */
+    public static class IdpConfig implements Parcelable {
```

Comment:
I tried this, but it makes parceling and passing a list of providers around much uglier.  I would prefer to keep it this way.